package.json

```diff
@@ -9,12 +9,12 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon -e ts --exec 'ts-node src/index.ts'",
+    "start": "nodemon -e ts --exec ts-node src/index.ts",
```

Comment:
@mac2000 Not 100% sure but recent versions of `nodemon` should support `nodemon src/index.ts`
Can you please test?

Reply:
![image](https://user-images.githubusercontent.com/88868/81306256-f003ff80-9087-11ea-8c10-88fc8f834c3f.png)

WoW indeed it did worked! Thank you did not know that